### PR TITLE
Add scale dash line option

### LINKED ISSUE #12: Add scaleDashLine to keep the scale on the dash line

### PR DESCRIPTION
**Pull request by clementlevasseur**
close #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,6 +147,15 @@ thickness at each scale (`false`).
 
 __NOTE:__ The canvas is automatically updated when this value is changed.
 
+### SnappyContext2D.scaleDashLine
+
+    ctx.scaleDashLine = <Boolean>
+
+Determines if the dash line is scaled (`true`, default) or if it keeps the same
```
Comment (flozz) [inline README.md]: Determines if the width of the dashes of dashed lines are scaled (`true`,
default) or if they keep the same width at each scale (`false`).

File: README.md [hunk 2]
```diff
@@ -58,6 +58,7 @@ ctx.render();
     globalTranslationX: <Number>,  // Initial x translation of the snappy context (default 0)
     globalTranslationY: <Number>,  // Initial y translation of the snappy context (default 0)
     scaleLineWidth: <Boolean>,     // If true, the snappy context will scale the line width according to scale (default true)
+    scaleDashLine: <Boolean>,      // If true, the snappy context will scale the dash line according to scale (default true)
```
Comment (flozz) [inline README.md]: // If true, the snappy context will scale the with of the dashes of dashed line according to scale (default true)

**Review by flozz — CHANGES_REQUESTED** [review]
Can you rename this option `scaleDashesWidth`, I think it is a more obvious name :)

File: README.md (hunk 1 above)
Comment (clementlevasseur) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (clementlevasseur) [reply README.md]: Done

**Review by flozz — APPROVED** [review]

**Pull request merged**